app/user/cached-user-resolver.js

```diff
@@ -0,0 +1,10 @@
+const userResolver = require('./user-resolver');
+const userInfoCache = require('../cache/cache-config').userInfoCache;
+const jwtUtil = require('../util/jwt');
+
+const getCachedUserDetails = (jwt) => {
+  return userInfoCache.get(jwtUtil.getJwt(jwt), 
+    () => userResolver.getUserDetails(jwt));
+};
+
+exports.getUserDetails = getCachedUserDetails;
```

Comment:
Not sure why,  if it get exposed with other name than getCachedUserDetails, it is not recognized and runtime. It comes as undefined. 
exports.getCachedUserDetails= getCachedUserDetails;

user-resolver  is doing exports.getTokenDetails = getTokenDetails;